Makefile

```diff
@@ -10,19 +10,17 @@ clean:
 
 .PHONY: lint
 lint:
-	isort --check-only evalml
 	sh ./import_or_skip.sh
 	python docs/notebook_version_standardizer.py check-versions
 	python docs/notebook_version_standardizer.py check-execution
-	black evalml docs/source -t py39 --check
-	pydocstyle evalml/ --convention=google --add-ignore=D107 --add-select=D400 --match-dir='^(?!(tests)).*'
-	flake8 evalml
+	black evalml/ docs/source/ --check --config pyproject.toml
+	ruff evalml/ docs/source/
```

Comment:
Have you been able to validate that this configuration catches all the style currently being enforced?

Reply:
Added pydocstyle check and fixes
https://github.com/alteryx/evalml/pull/3928/commits/98aff8f20d4050b9f809f713bad61434e1ffd755